msys2-runtime/0019-strace.cc-Don-t-set-MSYS-noglob.patch

```diff
@@ -50,12 +50,17 @@ index 0c8d75e..f7c98bd 100644
 +  See quoted () comment:
 +  "This must have been run from a Windows shell, so preserve
 +     quotes for globify to play with later."
-+
+ 
    const char *cygwin_env = getenv ("MSYS");
    const char *space;
-   if (cygwin_env)
-@@ -360,6 +378,7 @@ create_child (char **argv)
-   char *newenv = (char *) malloc (sizeof ("MSYS=noglob") + strlen (space) + strlen (cygwin_env));
+ 
+-  if (cygwin_env && strlen (cygwin_env) <= 256) /* sanity check */
++  if (cygwin_env && strlen (cygwin_env) <= 256) // sanity check
+     space = " ";
+   else
+     space = cygwin_env = "";
+@@ -364,6 +381,7 @@ create_child (char **argv)
+ 				  + strlen (space) + strlen (cygwin_env));
    sprintf (newenv, "MSYS=noglob%s%s", space, cygwin_env);
    _putenv (newenv);
 +*/
```

Comment:
@mati865 This is the original patch adding that end-of-comment: https://github.com/Alexpux/MSYS2-packages/blob/8302b167a80d184a23dc0bd00327d1ee040914b1/msys2-runtime/0019-strace.cc-Don-t-set-MSYS-noglob.patch#L61. And this is the patch after this PR, adding the same end-of-comment: https://github.com/dscho/MSYS2-packages/blob/update-msys2-runtime/msys2-runtime/0019-strace.cc-Don-t-set-MSYS-noglob.patch#L66

In other words, the hunk before this PR looks like this:

```diff
@@ -360,6 +378,7 @@ create_child (char **argv)
   char *newenv = (char *) malloc (sizeof ("MSYS=noglob") + strlen (space) + strlen (cygwin_env));
   sprintf (newenv, "MSYS=noglob%s%s", space, cygwin_env);
   _putenv (newenv);
+*/
   ret = CreateProcess (0, one_line.buf,	/* command line */
 		       NULL,	/* Security */
 		       NULL,	/* thread */
-- 
2.9.0
```

And afterwards:

```diff
@@ -364,6 +381,7 @@ create_child (char **argv)
 				  + strlen (space) + strlen (cygwin_env));
   sprintf (newenv, "MSYS=noglob%s%s", space, cygwin_env);
   _putenv (newenv);
+*/
   ret = CreateProcess (0, one_line.buf,	/* command line */
 		       NULL,	/* Security */
 		       NULL,	/* thread */
-- 
2.9.0
```

As you will appreciate, both hunks add the end of the comment in exactly the same place: after the `_putenv()` and before the `CreateProcess()`.